data_pipeline/config.json

```diff
@@ -1,6 +1,6 @@
 {
    "_comment": "coinpaprika ids @ https://api.coinpaprika.com/v1/coins",
-   "output":"/home/ubuntu/l2beat/treasury_optimism.json",
+   "output":"/data/output.json",
```

Comment:
nit: to make it obvious that it's relative path don't begin it with `/` but rather `./`